nautobot/extras/migrations/0116_create_job_queue_model.py

```diff
@@ -94,6 +94,18 @@ class Migration(migrations.Migration):
                 related_name="jobs", through="extras.JobQueueAssignment", to="extras.jobqueue"
             ),
         ),
+        migrations.AddField(
+            model_name="job",
+            name="default_job_queue_override",
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name="job",
+            name="default_job_queue",
+            field=models.ForeignKey(
+                blank=False, null=False, on_delete=django.db.models.deletion.PROTECT, to="extras.jobqueue"
+            ),
+        ),
```

Comment:
Does this actually work? I'd have thought we'd need to create the field as `blank=True, null=True`, populate it in the data migration, and only then change it to `blank=False, null=False`?

Reply:
Yep that's what Im bumping into in the CI I think, thank you!